core/kits/documents/tabs/layout-settings.php

```diff
@@ -126,7 +126,17 @@ public function register_tab_controls() {
 		 */
 		$page_templates_module = Plugin::$instance->modules_manager->get_modules( 'page-templates' );
 
-		$page_templates_module->add_template_controls( $this->parent, 'default_page_template', false );
+		$page_template_control_options = [
+			'label' => __( 'Default Page Layout', 'elementor' ),
+			'options' => [
+				// This is here because the "Theme" string is different than the default option's string.
+				'default' => __( 'Theme', 'elementor' ),
+			],
+		];
+
+		$page_template_control_options['options'] += $page_templates_module->add_page_templates( [], null, null );
```

Comment:
Extract to a variable at line 129. $options = ... + page_templates_module...

Reply:
@matipojo 

Like this?

```
$page_templates_module = Plugin::$instance->modules_manager->get_modules( 'page-templates' );
$page_templates = $page_templates_module->add_page_templates( [], null, null );

$page_template_control_options = [
	'label' => __( 'Default Page Layout', 'elementor' ),
	'options' => [
		// This is here because the "Theme" string is different than the default option's string.
		'default' => __( 'Theme', 'elementor' ),
		$page_templates,
	],
];
```

Reply:
Yes, but you need  `+` for merge the options arrays.

Reply:
Done